students/BrianNisonger/Session8/circle.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+
+class circle():
+    def __init__(self, radius):
+        self._radius = radius
+
+    @property
```

Comment:
no need for properties if you're not going to do anything in them. 

just store self.radius as a regular attribute.